tests/Dfe.EarlyYearsQualification.E2ETests/cypress.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = defineConfig({
     auth_secret: 'CX' // dummy value: pass in using Cypress command line --env auth_secret=an-acceptable-secret-value
   },
   e2e: {
-    baseUrl: "http://127.0.0.1:5025/",
+    baseUrl: "http://localhost:5025/",
```

Comment:
Didn't we have to set this to 127.0.0.1 to fix Firefox tests or has that been resolved now?

Reply:
I think it has been resolved, but I will move it to another PR just to not inflate this one